scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+Usage:
+    python scripts/download_cams_pm25.py --start 2024-01-01 --end 2024-01-31
+    python scripts/download_cams_pm25.py --start 2024-01-01 --end 2024-01-31 --forecast
+    python scripts/download_cams_pm25.py --locations Tokyo Osaka --start 2024-01-01
+"""
+
+import argparse
+import asyncio
+import sys
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)